static/app/views/alerts/rules/issue/addIntegrationRow.tsx

```diff
@@ -0,0 +1,116 @@
+import {useCallback, useEffect, useState} from 'react';
+import styled from '@emotion/styled';
+
+import {Button} from 'sentry/components/button';
+import PluginIcon from 'sentry/plugins/components/pluginIcon';
+import {space} from 'sentry/styles/space';
+import type {IntegrationProvider, Project} from 'sentry/types';
+import type {Organization} from 'sentry/types/organization';
+import useApi from 'sentry/utils/useApi';
+import {AddIntegrationButton} from 'sentry/views/settings/organizationIntegrations/addIntegrationButton';
+
+type Props = {
+  closeModal: () => void;
+  integrationSlug: string;
```

Comment:
Not 100% sure about this but I don't think integration slug is always the same as provider key and I believe what your code here gets is the provider key. For slack, msteams and discord they seem to be but it's not enforced anywhere in the code to be so I would rename it to avoid confusion.